src/libs/state.ts

```diff
@@ -259,9 +259,9 @@ export const workspacesStore = atom<WorkspaceWrapper[]>([]);
 
 export const rerunFailuresStatus = atom<unknown>(undefined);
 
-export const errorNotifiedRuntimes = atom<unknown[]>([]);
```

Comment:
In order to use a type-safe append, we need to the list and the appended value to be of the same value. These stores were having unknown[] but always adding a certain type.

Reply:
This appears to only be used in https://github.com/DataBiosphere/terra-ui/blob/3a5db7c5f0f09d00feae97d7ac0a84cbcdb65d55/src/analysis/AnalysisNotificationManager.ts